### PR TITLE
#123 Add Michelle to Website (Based On main)

### PR DESCRIPTION
**Pull request by michellemcgowan**
PR for Issue #119

### CODE REVIEW

File: TeamPage.html [hunk 1]
```diff
@@ -593,6 +593,48 @@ <h2>Meet Our Team</h2><br>
...
+                    <div class="modal-content">
+                      <div class="modal-header">
+                        <div class="modal-title"><p class="text-justify"><b>Michelle McGowan</b> - Student Programmer<br>
+                          Work Location: Remote<br>
+                          Email: michelle.mcgowan@unlv.edu</p>
+                        </div>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="close">x</button>
+                      </div>
+                      <div class="modal-body">
+                        <p class="text-justify">Michelle McGowan joined the UNLV-FIA group as a student intern in January 2023. She is currently pursuing a Bachelor of Science degree in Computer Science and hopes to graduate in the Spring of 2024. Michelle will be making sure that Section 508 standards are met on the DATIM site. In her spare time, Michelle enjoys reading, hanging out with friends and going to concerts.</p>
```
Comment (Kelvyn-Meyers) [inline TeamPage.html:615]: I believe your final sentence should contain a comma after "friends":
```
In her spare time, Michelle enjoys reading, hanging out with friends(,) and going to concerts.
```
But I could definitely be wrong about that, so please ignore this if so.

File: TeamPage.html [hunk 2]
```diff
@@ -593,6 +593,48 @@ <h2>Meet Our Team</h2><br>
                </div>
            </div>
           </div>
+          <div class="col-sm-3 col-xs" style="min-width: 210px;">
+            <div class="id_presentation">
+              <img class="img_staff" src="https://i.imgur.com/jxzYjQO.jpg" style="width: 200px; height: 200px; object-fit: cover; object-position: top;" alt="Michelle's Picture"/>
```
Comment (Kelvyn-Meyers) [inline TeamPage.html:598]: You might be able to center your picture a bit better if you remove the `object-position: top` CSS property here. But if this was intentional, let me know.

**Review by Kelvyn-Meyers — CHANGES_REQUESTED** [review]
Hi @michellemcgowan,

Nice work on your first official PR! I only have a couple of in-line comments below for potential fixes. Though one problem I noticed might be intentional and the other problem might not even be a problem at all. So if you don't think a change is warranted for any of them, just let me know in the comments before "Resolving" the issues below.

If you need any help on how to do that, please read my comments on PR #122.

Once you're ready, feel free to request my review again from the Requestor tab.

File: TeamPage.html (hunk 1 above)
Comment (michellemcgowan) [reply TeamPage.html]: Thank you! You are right!

**Review by Kelvyn-Meyers — APPROVED** [review]
Everything looks good, thank you for making those changes! Ready to merge.

**Pull request merged**